style/style.css

```diff
@@ -0,0 +1,312 @@
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Roboto", sans-serif;
+}
+
+body{
+    background-color: #ffffff;
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-template-rows: 200px auto auto;
+    grid-template-areas: 
+    "hd"
+    "section"
+    "footer";
+    gap: 1rem;
+}
+
+
+.header{
+    /* background-color: red; */
+    grid-area: hd;
+    background-image: url("/img/header\ pic.avif");
+    object-fit: cover;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+    width: 100%;
+}
+
+.nav-header {
+    display: none;
+}
+
+.add {
+    max-width: 24rem;
+    word-wrap: break-word;
+
+}
+
+.add-text {
+    color: rgb(255, 255, 255);
+    text-shadow: 0 0 20px #dae351, 0 0 10px #ff0000;
+    font-family: fantasy;
```

Comment:
It's fine to use custom fonts, but try to specify a fallback font as well, in case the custom font isn't available on the user's device.
eg `font-family: fantasy, Serif`
([more info](https://css-tricks.com/css-basics-fallback-font-stacks-robust-web-typography/))